### DISCUSSION TITLE
Log in one user to multiple oauth 2 clients

### DISCUSSION THREAD

**Discussion by FloRichard** [ory/hydra#2802] [Q&A] [answered]
Hello,

I have an issue but it is maybe due to a my miscomprehension of oauth 2.0 spec.
The question is : can i connect one user to two oauth 2 clients simultaneously ?
Because when i disconnect the user on the first client, the refresh token of the other client can't be found by Hydra like if it was deleted by the logout request.

Best Regards,

FloRichard.

**Comment by vinckr** [comment, marked as answer]
Hey Flo,
This should not be an issue, I have however not tested it. 
`when i disconnect the user on the first client, the refresh token of the other client can't be found by Hydra like if it was deleted by the logout request.`
Can you give me some steps to reproduce maybe? (create 2 clients ideally with the config you used, log in on both clients, log out on both clients(?), log back in or how exactly did you proceed?)
How are you running Hydra, which version etc.?

**Reply by FloRichard** [reply]
Thanks for your answer ! But i managed to deal with the issue. My server had big latencies so i think that the same refresh token was sent twice. Now that the latencies are gone, everything work as expected.

**Reply by vinckr** [reply]
Ok great, then it was most likely the latency 👍

**Reply by EiffelFly** [reply]
@FloRichard How do you connect two oauth client with one account at the first place? I am very interested at your process, due to I am facing some problem related to this issue `An account with the same identifier (email, phone, username, ...) exists already.`

Is the process something like, User A login with A Oauth client, then they can link the account with Client B. But what if User A register with Client A, but forgot and singin with client B, then he will face the issue of `An account with the same identifier (email, phone, username, ...) exists already.` How do you solve this kind of issue.

**Reply by FloRichard** [reply]
Hello @EiffelFly I think that we don't have the same use case. I'm using Hydra in a very specific way.
I'm inviting you to open a discussion in order to have better information :)
